.github/workflows/test-npgsql.yml

```diff
@@ -21,7 +19,7 @@ on:
 
   # Run job each night after CrateDB nightly has been published.
   schedule:
-    - cron: '0 2 * * *'
+    - cron: '0 3 * * *'
```

Comment:
The job for running the Npgsql tests already has been scheduled to run each night, at `0 2 * * *` cron time. At [^1], we can inspect the scheduled invocations.

The most recent one is indicated to have run at `11 hours ago`. It is [run #6687457412](https://github.com/crate/cratedb-examples/actions/runs/6687457412).

When activating timestamps on the log output, the first timestamp is `Mon, 30 Oct 2023 02:09:30 GMT`. So, it is actually GMT, and we are good to go?


[^1]: https://github.com/crate/cratedb-examples/actions/workflows/test-npgsql.yml